### PR TITLE
Upload basic German translation

### PR DESCRIPTION
**Pull request by Luffitys**

### CODE REVIEW

**Comment by peter9811** [comment]
@Luffitys 
I take yours translations and add here. Please check and try add more, if you want... 

https://crowdin.com/project/privatezilla-traduccin/de

File: src/Privatezilla/Properties/Resources.de.resx [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+    <value>Privatsphäre</value>
+  </data>
+  <data name="rootSettingsSecurity" xml:space="preserve">
+    <value>Sicherheit</value>
+  </data>
+  <data name="rootSettingsUpdates" xml:space="preserve">
+    <value>Updates</value>
+  </data>
+  <data name="LblStatus" xml:space="preserve">
+    <value>Drücke Analysieren um nach konfigurierten einstellungen zu suchen.</value>
```
Comment (hammy1988) [inline src/Privatezilla/Properties/Resources.de.resx:306]: Einstellungen

File: src/Privatezilla/Properties/Resources.de.resx [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+  </data>
+  <data name="PSBack" xml:space="preserve">
+    <value>Zurück</value>
+  </data>
+  <data name="PSImport" xml:space="preserve">
+    <value>Skript importieren</value>
+    <comment>Scripting menu</comment>
+  </data>
+  <data name="PSInfo" xml:space="preserve">
+    <value>Was macht diese/r Vorlage/Skript?\r\n</value>
```
Comment (hammy1988) [inline src/Privatezilla/Properties/Resources.de.resx:247]: diese/s

File: src/Privatezilla/Properties/Resources.de.resx [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+  <data name="statusSuccessApply" xml:space="preserve">
+    <value>Angewendet</value>
+  </data>
+  <data name="statusSuccessConfigure" xml:space="preserve">
+    <value>Konfiguriert</value>
+  </data>
+  <data name="TxtPSInfo" xml:space="preserve">
+    <value>Willkommen bei dem Modernen Richtlinien Editor, welcher Ihnen erlaubt, Gruppenrichtlinien und benutzerdefinierte Einstellungen in der Form von PowerShell Skripts und Vorgaben (gebündelte Skripts) anzuwenden.
+
+Wähle einen Skript aus, um seine Beschreibung zu sehen.
```
Comment (hammy1988) [inline src/Privatezilla/Properties/Resources.de.resx:223]: ein

**Pull request merged**